application/libraries/Email.php

```diff
@@ -10,4 +10,14 @@ public static function send($to, $subject, $message)
 
 			mail($to, $subject, $message, $headers);
 		}
+		
+		public static function receive($from, $subject, $message, $pseudo)
```

Comment:
On ajoute aussi un "Reply-To", et le premier argument de la fonction mail() n'est plus le même.
Alors je vois pas trop comment éviter les répétitions. Je peux ajouter 2 constantes mais je vois pas comment faire mieux.